index.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Dmitrij Krutalianau</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+    <base target="_blank">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div class="page">
+    <main>
+        <h1>Summary of Dima Krutolianov</h1>
+        <div class="container" id="introduction">
```

Comment:
What is the reason to use id here and below?

Reply:
@OlgaPoddubko, I wanted to implement a navigation bar, using these ids as anchors. But I had no time during the code jam and after it... Just forgot about this goal because of other more valuable tasks.

Reply:
@OlgaPoddubko,
1. Hm, I didn't know such semantics about class names. Thank you, I'll refactor it to use section elements instead.